operate/schema/src/main/java/io/camunda/operate/store/opensearch/client/sync/OpenSearchDocumentOperations.java

```diff
@@ -124,7 +137,8 @@ private <R> Map<String, Aggregate> scrollWith(
       scrollId = response.scrollId();
       List<Hit<R>> hits = response.hits().hits();
 
-      while (!hits.isEmpty() && scrollId != null) {
+      while (!hits.isEmpty() && scrollId != null && count < limit) {
+        count += response.hits().hits().size();
```

Comment:
All changes in this file should be reverted. We do not want to touch antyhing to do with the operate module

Reply:
fixed